docs/kqueen.rst

```diff
@@ -111,7 +113,7 @@ Example configuration files are located in `config/` directory. Default configur
       - Name of Jenkins job used to deploy cluster.
     * - JENKINS_USERNAME
       - None
-      - Username for Jenkins login.
+      - Optional. Default username for Jenkins login. Is not used in UI
 
     * - CLUSTER_ERROR_STATE
```

Comment:
Caption for a cluster in error state.
Caption for a cluster in OK state.
Caption for a cluster in provisioning state.
Caption for a cluster in deprovisioning (deleting) state.
Caption for a cluster with unknown state.